python/docs/source/development/ps_contributing.rst

```diff
@@ -0,0 +1,192 @@
+==================
+Contributing Guide
+==================
+
+.. contents:: Table of contents:
+   :depth: 1
+   :local:
+
+Types of Contributions
+======================
+
+The largest amount of work consists simply of implementing the pandas API using Spark's built-in functions, which is usually straightforward. But there are many different forms of contributions in addition to writing code:
+
+1. Use the project and provide feedback, by creating new tickets or commenting on existing relevant tickets.
+
+2. Review existing pull requests.
+
+3. Improve the project's documentation.
+
+4. Write blog posts or tutorial articles evangelizing Koalas and help new users learn Koalas.
```

Comment:
So, are we planning to rename the 'Koalas' stuffs to 'pandas API on Spark' in the separated PR ??

Reply:
yeah, let's create JIRAs and work on that.

Reply:
Yes, just created: https://issues.apache.org/jira/browse/SPARK-35591